.github/workflows/containers.yml

```diff
@@ -103,10 +103,18 @@ jobs:
                     APIML_SECURITY_OIDC_CLIENTSECRET: ${{ secrets.OKTA_CLIENT_PASSWORD }}
                     APIML_SECURITY_OIDC_INTROSPECTENDPOINT: /v1/introspect
                     APIML_SECURITY_OIDC_ENABLED: true
+                    APIML_SECURITY_OIDC_REGISTRY: zowe.okta.com
+                    APIML_SECURITY_X509_EXTERNALMAPPERUSER: mockUser
+                    APIML_SECURITY_X509_EXTERNALMAPPERURL: https://mock-services:10013/zss/api/v1/certificate
             discovery-service:
                 image: ghcr.io/balhar-jakub/discovery-service:${{ github.run_id }}-${{ github.run_number }}
             mock-services:
                 image: ghcr.io/balhar-jakub/mock-services:${{ github.run_id }}-${{ github.run_number }}
+            discoverable-client:
+                image: ghcr.io/balhar-jakub/discoverable-client:${{ github.run_id }}-${{ github.run_number }}
+                env:
+                    APIML_SERVICE_AUTHENTICATION_SCHEME: zosmf
```

Comment:
there are multiple statically registered services, can we use some of them? Or register another with zosmf scheme

Reply:
duplicate

Reply:
Resolved. All static definitions of discoverable client are now leveraged.